src/components/GrampsjsPerson.js

```diff
@@ -49,16 +49,17 @@ export class GrampsjsPerson extends GrampsjsObject {
     }
     const surname = this.data.profile.name_surname || html`&hellip;`
     const suffix = this.data.profile.name_suffix || ''
+    const nick = this.data?.primary_name?.nick
     let given = this.data.profile.name_given || html`&hellip;`
     const call = this.data?.primary_name?.call
     const callIndex = call ? given.search(call) : -1
     given =
       callIndex > -1
-        ? html`${given.substr(0, callIndex)}<span class="given-name"
-              >${given.substr(callIndex, call.length)}</span
-            >${given.substr(callIndex + call.length)}`
+        ? html`${given.substring(0, callIndex)}<span class="given-name"
+              >${given.substring(callIndex, call.length)}</span
+            >${given.substring(callIndex + call.length)}`
         : given
-    return html`${given} ${surname} ${suffix}`
+    return html`${given} ${nick ? `"${nick}" ` : ''}${surname} ${suffix}`
```

Comment:
Would be nice to use typographical quotation marks (although strictly speaking that would even by locale dependent)

Reply:
But on second thought I am not sure I would like to always have the nickname displayed as part of the page title ... does Gramps Desktop do it like that? I think we can use that as guidance to have a more consistent user experience. For instance, if both Gramps & Gramps Web would display nicknames as part of the person profile's title, I would likely want to move more informal nick names to a secondary name to prevent them from showing up...

Reply:
How do you deal with those kind of suggestions in the project? Should I open a discussion here in gitlab?

But you are right. I guess the best thing would be, if the user could choose (and define it in the user settings). 

The scenario I came up with this was: It used to be like that, that parents and children often share the same given name. Sometimes they differ in second or third given names (which might not be known). To make it easier to distinguish persons, this might help.

I will remove this for now.

In my family tree there are some persons with this issue 😄 This is why they mostly have a nickname. To distinguish e.g. between mother and daughter.

Reply:
Maybe I can implement this when solving https://github.com/gramps-project/gramps-web/issues/67